### ISSUE TITLE
Can't startup: FastlaneBuildRunner undefined method `user' for nil:NilClass 

### ISSUE THREAD

**Issue by taquitos** [fastlane/ci#941]
```
[2018-05-28 22:41:16] DEBUG FastlaneBuildRunner:  Setting build status to failure due to exception
[2018-05-28 22:41:16] ERROR FastlaneBuildRunner:  undefined method `user' for nil:NilClass
[2018-05-28 22:41:16] ERROR FastlaneBuildRunner:  /Users/jliebowitz/dev/ci/app/services/xcode_manager_service.rb:33:in `use_apple_id'
/Users/jliebowitz/dev/ci/app/services/xcode_manager_service.rb:22:in `initialize'
/Users/jliebowitz/dev/ci/app/services/services.rb:191:in `new'
/Users/jliebowitz/dev/ci/app/services/services.rb:191:in `xcode_manager_service'
/Users/jliebowitz/dev/ci/app/features/build_runner/fastlane_build_runner.rb:170:in `block in run'
/Users/jliebowitz/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/bundler-1.16.1/lib/bundler/shared_helpers.rb:65:in `chdir'
/Users/jliebowitz/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/bundler-1.16.1/lib/bundler/shared_helpers.rb:65:in `block in chdir'
/Users/jliebowitz/.rbenv/versions/2.3.3/lib/ruby/2.3.0/monitor.rb:214:in `mon_synchronize'
/Users/jliebowitz/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/bundler-1.16.1/lib/bundler/shared_helpers.rb:64:in `chdir'
/Users/jliebowitz/dev/ci/app/features/build_runner/fastlane_build_runner.rb:128:in `run'
/Users/jliebowitz/dev/ci/app/features/build_runner/build_runner.rb:289:in `run_action'
/Users/jliebowitz/dev/ci/app/features/build_runner/build_runner.rb:310:in `block (2 levels) in start'
/Users/jliebowitz/dev/ci/app/features/build_runner/build_runner.rb:178:in `block in pre_run_action'
/Users/jliebowitz/dev/ci/app/shared/models/git_repo.rb:507:in `block in checkout_commit'
/Users/jliebowitz/dev/ci/app/shared/models/git_repo.rb:446:in `block in perform_block'
/Users/jliebowitz/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/bundler/gems/TaskQueue-5b3f459be5c4/lib/queue_worker.rb:82:in `block in process'
```

**Comment by taquitos** [comment]
This must be due to a env var or something, still catching up from vacation 🏖

**Comment by taquitos** [comment]
Yup, old bug, missing env var. Will be fixed in #870
